plugins/modules/azure_rm_storageaccount.py

```diff
@@ -822,12 +821,6 @@ def create_account(self):
             if self.blob_cors:
                 account_dict['blob_cors'] = self.blob_cors
             return account_dict
-        if bool(self.https_only):
-            self.https_only = False
-        if bool(self.minimum_tls_version):
-            self.minimum_tls_version = 'TLS1_0'
-        if bool(self.allow_blob_public_access):
```

Comment:
if self.allow_blob_public_access is None:
            self.allow_blob_public_access = True